sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Generated/Models/DatabaseAccountCreateUpdateProperties.cs

```diff
@@ -19,7 +19,6 @@ namespace Microsoft.Azure.Management.CosmosDB.Models
     /// <summary>
     /// Properties to create and update Azure Cosmos DB database accounts.
     /// </summary>
-    [Newtonsoft.Json.JsonObject("DatabaseAccountCreateUpdateProperties")]
```

Comment:
I wonder why this got removed. It may have to do with the autorest version used to generate the package.

Reply:
What was the command you used to generate the package ?

Reply:
Please note this is not a blocking comment if you cant find the reason to this change,, feel free to ignore it.

Reply:
i'm not sure what that got removed..

this is the autorest command that i used:
autorest.cmd "C:\code\swagger\azure-rest-api-specs\specification\cosmos-db\resource-manager\readme.md" --csharp --version=v2 --reflect-api-versions --csharp-sdks-folder=C:\code\sdk\azure-sdk-for-net\sdk